src/components/Checkbox/Checkbox.styl

```diff
@@ -0,0 +1,23 @@
+.container {
+	display: inline-block;
+	position: relative;
+
+	input[type="checkbox"] {
+		position: absolute;
+		clip: rect(0,0,0,0);
+	}
+
+	&__checkboxIcon {
```

Comment:
using css-modules you can get rid of these BEM-ish nesting, just use plain names